reportportal/values.yaml

```diff
@@ -35,9 +35,16 @@ uat:
       memory: 2048Mi
   sessionLiveTime: 86400
   podAnnotations: {}
-  jvmArgs: "-Djava.security.egd=file:/dev/./urandom -XX:MinRAMPercentage=60.0 -XX:MaxRAMPercentage=90.0"
   securityContext: {}
   serviceAccountName: ""
+  jvmArgs: "-Djava.security.egd=file:/dev/./urandom -XX:MinRAMPercentage=60.0 -XX:MaxRAMPercentage=90.0"
+  #jvmArgs: "-Djavax.net.ssl.trustStore=/etc/secret-volume/custom-pki.jks -Djava.security.egd=file:/dev/./urandom -XX:MinRAMPercentage=60.0 -XX:MaxRAMPercentage=90.0"
+  secret:
+    enabled: false
+    mountPath: /etc/secret-volume
+    readOnly: true
+    data: {}
+     #custom-pki.jks: <base64-data>
```

Comment:
Please use common comments style: 2 space indent, a space after has sign.

Reply:
Done through this commit : https://github.com/reportportal/kubernetes/pull/232/commits/11c57743c0b2e71261171af9ae083e51d11c79e0